composer.json

```diff
@@ -21,7 +22,7 @@
         "guzzlehttp/psr7": "^1.7"
     },
     "suggest": {
-        "ext-libsodium": "Provides faster verification of updates"
+        "ext-sodium": "Provides faster verification of updates"
```

Comment:
`ext-libsodium` was _never_ correct, as it turns out. `composer show --platform` shows that the correct extension name is `ext-sodium`, and the _library_ name is `lib-libsodium`. (Not sure what the difference is, exactly, but there is definitely no `ext-libsodium`.)